apps/prairielearn/src/lib/question-render.js

```diff
@@ -696,41 +637,34 @@ export async function renderPanelsForSubmission({
         question_course,
         locals,
       );
-      const grading_job_stats = buildGradingJobStats(grading_job);
 
       panels.answerPanel = locals.showTrueAnswer ? htmls.answerHtml : null;
       panels.extraHeadersHtml = htmls.extraHeadersHtml;
 
       await manualGrading.populateRubricData(locals);
       await manualGrading.populateManualGradingData(submission);
-      const renderParams = {
-        course: question_course,
-        course_instance,
+
+      panels.submissionPanel = SubmissionPanel({
+        questionContext,
         question,
+        variant_id: variant.id,
+        assessment_question,
+        instance_question,
+        course_instance_id: course_instance?.id,
         submission: /** @type {SubmissionForRender} */ ({
```

Comment:
Is this type annotation still necessary now that `SubmissionPanel` presumably includes a type for `submission`?

Reply:
Not necessary. Removed.